src/WopiValidator/Options.cs

```diff
@@ -20,6 +20,12 @@ class Options
 		[Option('l', "token_ttl", Required = true, HelpText = "WOPI access token ttl")]
 		public long AccessTokenTtl { get; set; }
 
+		[Option('r', "X-WOPI-UsingRestrictedScenario", Required = false, HelpText ="Restricted scenario")]
+		public string UsingRestrictedScenario { get; set; }
+
+		[Option('a', "X-WOPI-ApplicationId", Required = false, HelpText = "Id of an application stored in secure store")]
+		public string ApplicationId { get; set; }
```

Comment:
`[Option('a', "ApplicationId", Required = false, HelpText = "ID of an application stored in secure store; applies only to the ReadSecureStore operation")]`